src/components/Table/BaseTable.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  Cell,
+  ExpandedContent,
+  flexRender,
+  HeaderCell,
+  HeaderRow,
+  type LGRowData,
+  Row,
+  Table,
+  TableBody,
+  type TableProps,
+  TableHead,
+} from "@leafygreen-ui/table/new";
+
+type SpruceTableProps = {
+  "data-cy-row"?: string;
+  "data-cy-table"?: string;
+};
+
+export const BaseTable = <T extends LGRowData>({
+  "data-cy-row": dataCyRow,
+  "data-cy-table": dataCyTable,
+  table,
+  ...args
+}: SpruceTableProps & TableProps<T>) => (
+  <Table data-cy={dataCyTable} table={table} {...args}>
+    <TableHead>
+      {table.getHeaderGroups().map((headerGroup) => (
+        <HeaderRow key={headerGroup.id}>
+          {headerGroup.headers.map((header) => (
+            <HeaderCell key={header.id} header={header}>
+              {flexRender(header.column.columnDef.header, header.getContext())}
+            </HeaderCell>
+          ))}
+        </HeaderRow>
+      ))}
+    </TableHead>
+    <TableBody>
+      {table.getRowModel().rows.map((row) => (
```

Comment:
There's a ticket for LG to fix this! I think it would be fairly difficult to do in Spruce so we should probably wait on them. [LG-3245](https://jira.mongodb.org/browse/LG-3245)